configure/Dockerfile.ubuntu

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:latest
+
+RUN apt-get update && apt-get install -y curl
+
+WORKDIR "/coder"
```

Comment:
Why change the workdir instead of doing `COPY "configure" "/coder/configure"`?

Reply:
No major reason other than how I've done things before. I'll push that change!